plugins/rapid7_insightidr/unit_test/test_query.py

```diff
@@ -4,8 +4,75 @@
 sys.path.append(os.path.abspath("../"))
 
 from unittest import TestCase
+from unittest.mock import patch
+from komand_rapid7_insightidr.actions.query.action import Query
+from komand_rapid7_insightidr.actions.query.schema import Input
+from komand_rapid7_insightidr.connection.schema import Input as ConnectionInput
+from insightconnect_plugin_runtime.exceptions import PluginException
+from unit_test.util import Util
+from unit_test.mock import mock_get_request
 
 
 class TestQuery(TestCase):
-    def test_query(self):
-        pass
+    @classmethod
```

Comment:
Can you run coverage to find out what percentage of the code is being tested here?